POSTINSTALL-README.md

```diff
@@ -95,3 +95,13 @@ The custom rule allows the group`diskadmin` to do the actions for unlocking and
 ## Instruction set optimizations for hardened_malloc
 
 Please see the description for release [v2.2.0](https://github.com/secureblue/secureblue/releases/tag/v2.2.0)
+
+## Enable Usbguard
+
+1. ```sudo sh -c 'usbguard generate-policy > /etc/usbguard/rules.conf'```
+2. ```sudo systemctl start usbguard.service```
+3. ```sudo systemctl enable usbguard.service```
+4. ```sudo systemctl start usbguard-dbus.service```
+5. ```sudo systemctl enable usbguard-dbus.service```
```

Comment:
This needs to be built in to the image, not a postinstall step. Postinstall steps are for things we can't automate

Reply:
The policy generation stuff should be in a ujust command, probably. And enabling the service should be automated.